pkg/manager/member/tiflash_member_manager.go

```diff
@@ -146,7 +146,7 @@ func (tfmm *tiflashMemberManager) syncHeadlessService(tc *v1alpha1.TidbCluster)
 	if !equal {
 		svc := *oldSvc
 		svc.Spec = newSvc.Spec
-		err = controller.SetServiceLastAppliedConfigAnnotation(newSvc)
+		err = controller.SetServiceLastAppliedConfigAnnotation(&svc)
```

Comment:
Do we need to set clusterIP here?

Reply:
We support setting clusterIP in the `service` of tc, so only need to set the clusterIP if it's not set in the `service` of the tc.